### PR TITLE
output sequence length rather than sequence

### PR DESCRIPTION
**Pull request by rzlim08**
* The python csv module was breaking because we were outputting the sequence and it was too long
* Use `awk` to just output the sequence length

### CODE REVIEW

File: workflows/long-read-mngs/run.wdl [hunk 1]
```diff
@@ -308,7 +308,7 @@ task RunReadsToContigs {
 
         # convert non-contigs.fastq file to .fasta file
         seqtk seq -a sample.non_contigs.fastq > sample.non_contigs.fasta
-        samtools view -F 0x900 sample.reads_to_contigs.sam | grep -v "^@" | cut -f1,3,10 > reads_to_contigs.tsv
+        samtools view -F 0x900 sample.reads_to_contigs.sam | grep -v "^@" | awk '{ print $1 "\t" $3 "\t" length($10)}' > reads_to_contigs.tsv
```
Comment (robertaboukhalil) [inline workflows/long-read-mngs/run.wdl:311]: Curious: are you using `grep -v` to remove the SAM header lines? If so, one alternative is to use the `--no-header` flag: `samtools view --no-header -F 0x900 sample.reads_to_contigs.sam`

**Review by robertaboukhalil — APPROVED** [review]
One minor suggestion, otherwise looks good to me!

File: workflows/long-read-mngs/run.wdl (hunk 1 above)
Comment (robertaboukhalil) [reply workflows/long-read-mngs/run.wdl:311]: Oh actually, according to [the docs](http://www.htslib.org/doc/samtools-view.html#OPTIONS), `--no-header` is the default, so you might be able to get away with just remove grep

Comment (rzlim08) [reply workflows/long-read-mngs/run.wdl]: thanks! I think this was a leftover from when were just `cat`-ing the output.

**Pull request merged**